compute/cloud-client/src/test/java/compute/disks/HyperdisksIT.java

```diff
@@ -58,34 +57,36 @@ public static void requireEnvVar(String envVarName) {
   }
 
   @BeforeAll
-  public static void setUp() {
+  public static void setUp()
+      throws IOException, ExecutionException, InterruptedException, TimeoutException {
     requireEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
     requireEnvVar("GOOGLE_CLOUD_PROJECT");
-
     HYPERDISK_NAME = "test-hyperdisk-enc-" + UUID.randomUUID();
     HYPERDISK_IN_POOL_NAME = "test-hyperdisk-enc-" + UUID.randomUUID();
     STORAGE_POOL_NAME = "test-storage-pool-enc-" + UUID.randomUUID();
+
+    Util.cleanUpExistingDisks("test-hyperdisk-enc-", PROJECT_ID, ZONE);
   }
 
   @AfterAll
   public static void cleanup()
        throws IOException, InterruptedException, ExecutionException, TimeoutException {
     // Delete all disks created for testing.
-    DeleteDisk.deleteDisk(PROJECT_ID, ZONE_1, HYPERDISK_NAME);
-    //    DeleteDisk.deleteDisk(PROJECT_ID, ZONE_2, HYPERDISK_IN_POOL_NAME);
-    //
-    //    try (StoragePoolsClient client = StoragePoolsClient.create()) {
-    //      client.deleteAsync(PROJECT_ID, ZONE_2, STORAGE_POOL_NAME);
-    //    }
+    DeleteDisk.deleteDisk(PROJECT_ID, ZONE, HYPERDISK_NAME);
+    //DeleteDisk.deleteDisk(PROJECT_ID, ZONE, HYPERDISK_IN_POOL_NAME);
+
+    //try (StoragePoolsClient client = StoragePoolsClient.create()) {
+    //  client.deleteAsync(PROJECT_ID, ZONE, STORAGE_POOL_NAME);
+    //}
```

Comment:
Why is this commented out?

Reply:
I tried to resolve issue with HyperdiskIT, but had an issue deleting Storage Pool. The cleanup method for this was not created before. I will create a new ticked for this. Now it commented to move forward tasks for reservation.

Reply:
Created method to delete StoragePool in Util class. Implemented cleanup method for StoragePool.